helmfile/helmfile.yaml

```diff
@@ -14,13 +14,13 @@ environments:
 
 releases:
 # Single Tier Website
-  - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
-    chart: ../helm/Charts/single-tier-appliation
+  - name: {{ requiredEnv "PROJECT" }}-{{ .Environment.Name }}
+    chart: ../helm/charts/single-tier-application
     values:
       - overrides/eligibility-estimator-client/eligibility-estimator-client.yaml.gotmpl
```

Comment:
would it be okay if I renamed these paths (and also updated any references elsewhere), or should I stick with it as is? I'd like to use `coabe` instead of `eligibility-estimator-client` if possible. I can make the change and push it here if that's cool!

Reply:
This is 100% up to you!

Reply:
I have pushed a commit renaming references from `eligibility-estimator-client` to `coabe`. I applied this in some places I'm not sure if I should have, specifically within `helm/charts/single-tier-application/Chart.yaml` and `values.yaml`. I decided to leave `single-tier-application` as-is rather than rename to `coabe` for now as I wasn't sure if that made sense to do or not. I figured "single tier application" represented the type of app it was, rather than the name of the app itself, but I guess it doesn't matter much anyways.

Either way, I hope this still works, let me know if I'm okay to merge it in or not, and if there's anything else I should change. Note that I plan on pulling in the changes from the template in a separate PR - I wanted to keep this PR's scope limited.

Reply:
Hoping that these changes are okay, and proceeding with the merge.